src/main/java/br/com/sbk/sbking/gui/elements/GameNameElement.java

```diff
@@ -0,0 +1,34 @@
+package br.com.sbk.sbking.gui.elements;
+
+import static javax.swing.SwingConstants.CENTER;
+
+import java.awt.Container;
+import java.awt.Font;
+import java.awt.Point;
+import java.awt.event.ActionListener;
+
+import javax.swing.JLabel;
+
+import br.com.sbk.sbking.gui.constants.FontConstants;
+import br.com.sbk.sbking.gui.constants.FrameConstants;
+import br.com.sbk.sbking.gui.jelements.SBKingLabel;
+
+public class GameNameElement {
+
+    private static final int DEFAULT_FONT_SIZE = 40;
+    private static final int DEFAULT_WIDTH = 500;
+    private static double scaleFactor;
+
+    public GameNameElement(String gameName, Container container, Point point, ActionListener actionListene) {
+        scaleFactor = FrameConstants.getScreenScale();
+        int fontSize = (int) (scaleFactor * DEFAULT_FONT_SIZE);
+
+        JLabel gameNameLabel = new SBKingLabel(gameName);
+        gameNameLabel.setSize((int) (DEFAULT_WIDTH * scaleFactor), (int) (fontSize * 1.1));
+        gameNameLabel.setLocation(point);
```

Comment:
This should include some form of centralization. I guess a `point.translate(-width/2, 0);` would be enough. Also `Point point`should have a better name, like `Point position` or `Point center`.